frontend/app/src/components/helper/NftDetails.vue

```diff
@@ -44,7 +44,7 @@ const loading = isLoading(Section.NON_FUNGIBLE_BALANCES);
 
 const { isPending } = useAssetCacheStore();
 const isNftDetailLoading: ComputedRef<boolean> = computed(
-  () => get(loading) || get(isPending(get(identifier)))
+  () => get(loading) || get(isPending(identifier))
```

Comment:
In my open PR I will be introducing https://vueuse.org/functions.html#category=%40Math so that we can use the `logicOr`/ `logicAnd` etc between refs